### PR TITLE
portcullis 1.1.0

### PR DESCRIPTION
**Pull request by maplesond**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/portcullis/1.1.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "portcullis" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "872c0dbd7515229ecc22c9bdcd72eb78dfe93a3c0bfd14af52c448c142fe892a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+   fn: Release-{{version}}.tar.gz
```
Comment (druvus) [inline recipes/portcullis/1.1.0/meta.yaml]: Please change it to `portcullis-{{version}}.tar.gz`

File: recipes/portcullis/1.1.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+about:
+  home: https://github.com/maplesond/portcullis
+  license: GPL3
+  license_file: COPYING
+  summary: 'Splice junction analysis and filtering from BAM files'
+
+build:
+  number: 1
+  skip: True # [osx or py27 or py34]
```
Comment (druvus) [inline recipes/portcullis/1.1.0/meta.yaml]: Bioconda is not building for py34 so it can be removed

File: recipes/portcullis/1.1.0/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+   sha256: {{ sha256 }}
+
+about:
+  home: https://github.com/maplesond/portcullis
+  license: GPL3
+  license_file: COPYING
+  summary: 'Splice junction analysis and filtering from BAM files'
+
+build:
+  number: 1
```
Comment (druvus) [inline recipes/portcullis/1.1.0/meta.yaml]: New recipes should start at 0

**Comment by maplesond** [comment]
Thanks very much for the comments @druvus.  I've made the changes and pushed up the changes.

**Comment by daler** [comment]
@maplesond I just sent an invite to be added to the bioconda team; once you join, the tests should use the bioconda mac OS plan.

**Comment by maplesond** [comment]
Many thanks @daler.  I have just joined up now.

**Comment by maplesond** [comment]
Hi @daler.  I was looking for a way to restart the macos build, but it says I still don't have a Mac plan.  Any suggestions?

**Comment by daler** [comment]
@maplesond a lot of people are having similar issues, I need to write some docs we can refer to . . . for now can you try closing this PR and opening a new one (maybe on a different branch)?

**Comment by daler** [comment]
(sorry about all the trouble)

**Comment by maplesond** [comment]
No worries.  Created PR #8321 with squashed commits on a different branch.

**Pull request closed without merging**